community/item-schema.json

```diff
@@ -1,280 +1,338 @@
 {
```

Comment:
Can you change the formatting back to how it was? Currently your PR changes every line in the item schema, which makes it hard to tell what actually changed.